slurm-on-gke/README.md

```diff
@@ -0,0 +1,445 @@
+
+Another best practice is to tag your images with the NVIDIA driver version that you will install. Doing so will help you manage different images because the tags match the NVIDIA driver that’s installed onto your GKE cluster.
+
+## Notes and requirements
+
+The current version of this guide does **not** address running the login pod with users different from root.  
+The following are required to be present, configured or available:
+
+* terraform v1.9.3 or newer  
+* docker 20.10.21 or newer  
+* kubectl  
+* A Google Cloud organization
+
+## Infrastructure and GKE
+
+This section describes the steps for creating and storing the container image.
+
+### Set up your environment
+
+In this tutorial, you use [Cloud Shell](https://cloud.google.com/shell) to manage resources that are hosted on Google Cloud. Cloud Shell is preinstalled with the software that you need for this tutorial, including [`kubectl`](https://kubernetes.io/docs/reference/kubectl/), the [Google Cloud CLI](https://cloud.google.com/sdk/gcloud), [Helm](https://helm.sh/), and [Terraform](https://cloud.google.com/docs/terraform).
+
+To set up your environment with Cloud Shell, follow these steps:
+
+1\. In the [Google Cloud console](http://console.cloud.google.com), launch a Cloud Shell session by clicking *Activate Cloud Shell*. This launches a session in the bottom pane of the Google Cloud console.
+
+2\. Set environment variables.
+
+```bash
+export PROJECT_ID=YOUR_PROJECT_ID
+export REGION=europe-west3 
+```
+
+Replace *YOUR_PROJECT_ID* with your Google Cloud [project ID](https://cloud.google.com/resource-manager/docs/creating-managing-projects#identifying_projects).
+
+3\. Set the default environment variables.
+
+```bash
+gcloud config set project ${PROJECT_ID}
+```
+
+4\. Clone the code repository.
+
+```bash
+git clone https://github.com/GoogleCloudPlatform/ai-on-gke
+```
+
+### Create your cluster infrastructure
+
+In this section, you run a Terraform script to create a private, highly available, regional GKE cluster.
+
+The Terraform module will also create a new project and anything that might be needed to set up the environment described in this guide. If you already have a GKE cluster where you can test the Slurm installation, you can skip this step.
+
+5\. Initialize Terraform.
+
+```bash
+cd slurm-on-gke
+cd infrastructure 
+terraform init
+```
+
+6\. Create the `terraform.tfvars` file with your own values.
+
+A `terraform.tfvars` file should be provided with all the values for the required variables. In the file, enter your own values as follows:
+
+```HCL
+impersonate_service_account = "YOUR_SERVICE_ACCOUNT_ID" <-- if you are using one
+region                      = "europe-west3"
+project_id                  = "YOUR_PROJECT_ID"
+billing_account_id          = "YOUR_BILLING_ACCOUNT_ID"
```

Comment:
We generally expect users of ai-on-gke to bring an existing project, would that remove the need to specify a billing account ID and folder ID?

Reply:
This also has implications for how easy it is to use the guide, many people trying the slurm guide may not even know the billing ID and folder ID, but they may have an existing project to work with